### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sanchita**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,16 +21,30 @@ with:
...
+
+```yaml
+uses: obvious/weaver@v0.0.1
+with:
+  tokens_path: '/Users/tokens.json'
+  output_path: 'output'
+```
+
+
+
```
Comment (sanchita) [inline README.md]: @msasikanth  Need your help to update a sample path here.

File: README.md [hunk 2]
```diff
@@ -83,12 +102,18 @@ the following command to create the `dist` folder with required files.
...
-## Pending Tasks
+## Pending Features
 
 - [ ] Typography support for iOS
+
 - [ ] Gradient colors support
+
 - [ ] Jetpack Compose support
 
+
```
Comment (sanchita) [inline README.md]: Do we need to add the License here? I am not sure if we can add it now but maybe a line redirecting to the LICENSE file?

File: README.md [hunk 3]
```diff
@@ -55,8 +73,9 @@ core/ios
 
 ### Pre-requisites
 
-Project requires NodeJS and NPM to build. If you already have Node installed, skip this step. If not
-you can download the [Node JS]
+Project requires NodeJS and `npm` to build. 
+
+If you already have Node installed, skip this step. If not you can download the [Node JS]
```
Comment (sanchita) [inline README.md]:
```suggestion
If you already have Node installed, skip this step. If not you can download [Node JS]
```

File: README.md [hunk 4]
```diff
@@ -4,10 +4,14 @@
 ![CHECK-DIST](https://github.com/obvious/weaver/actions/workflows/check_dist.yml/badge.svg)
 [![Code Style: Google](https://img.shields.io/badge/code%20style-google-blueviolet.svg)](https://github.com/google/gts)
 
-Weaver is a [GitHub Action] to transform the [Figma Token Studio] tokens to platform specific theme files
+Weaver is a [GitHub Action] that enables syncing of tokens from your design system Figma to code. It automates the process of updating your design system tokens in code each time a Figma token is updated. Hence, reducing developer intervention needed in maintaining tokens across platforms. 
+
+The action integrates with Figma [Token Studio](https://www.figma.com/community/plugin/843461159747178978/Tokens-Studio-for-Figma-(Figma-Tokens)) plugin to transform the token's JSON the plugin creates. The action supports Android and iOS platforms. 
 
 ## Usage
```
Comment (sanchita) [inline README.md]: @msasikanth I suppose we need a setup for Tokens Studio to link to Weaver. Don't we?

File: README.md [hunk 5]
```diff
@@ -4,10 +4,14 @@
 ![CHECK-DIST](https://github.com/obvious/weaver/actions/workflows/check_dist.yml/badge.svg)
 [![Code Style: Google](https://img.shields.io/badge/code%20style-google-blueviolet.svg)](https://github.com/google/gts)
 
-Weaver is a [GitHub Action] to transform the [Figma Token Studio] tokens to platform specific theme files
+Weaver is a [GitHub Action] that enables syncing of tokens from your design system Figma to code. It automates the process of updating your design system tokens in code each time a Figma token is updated. Hence, reducing developer intervention needed in maintaining tokens across platforms. 
+
+The action integrates with Figma [Token Studio](https://www.figma.com/community/plugin/843461159747178978/Tokens-Studio-for-Figma-(Figma-Tokens)) plugin to transform the token's JSON the plugin creates. The action supports Android and iOS platforms.
```
Comment (sanchita) [inline README.md]:
```suggestion
The action integrates with [Figma Token Studio] plugin to transform the token's JSON the plugin creates. The action supports Android and iOS platforms. 
```

File: README.md [hunk 6]
```diff
@@ -17,16 +21,30 @@ with:
...
+  tokens_path: '/Users/tokens.json'
+  output_path: 'output'
+```
+
+
+
 #### Expected JSON Structure
 
-You can find the sample JSON strucutre we are using in [sample_tokens] folder. We have a `core` file which contains
+You can find the sample JSON structure we are using in [sample_tokens] folder. We have a `core` file which contains
```
Comment (sanchita) [inline README.md]:
```suggestion
You can find a sample JSON structure in [sample_tokens] folder. We have a `core` file which contains
```

File: README.md (hunk 2 above)
Comment (msasikanth) [reply README.md]: Yeah, we can add it now as well

File: README.md [hunk 7]
```diff
@@ -83,12 +102,18 @@ the following command to create the `dist` folder with required files.
...
 - [ ] Typography support for iOS
+
 - [ ] Gradient colors support
+
 - [ ] Jetpack Compose support
 
+  
+
+## References 
+
```
Comment (msasikanth) [inline README.md]: We need to remove this, this just adds an empty section.

File: README.md (hunk 4 above)
Comment (msasikanth) [reply README.md]: Yeah, it will be useful to have, since we only support that JSON output at the moment.

**Pull request merged**